tests/_test_msui/test_mscolab.py

```diff
@@ -737,6 +804,9 @@ def assert_operation_is_listed():
         qtbot.wait_until(assert_operation_is_listed)
 
     def _activate_operation_at_index(self, index):
+        # The main window must be on top
+        QtTest.QTest.keyClick(self.window, QtCore.Qt.Key_Up, QtCore.Qt.ControlModifier)
```

Comment:
This line is a bit cryptic and I don't know if this would be another potential race condition. In my mental model of Qt every keyClick is always an asynchronous event, so I think it would be.

Could this be replaced by https://doc.qt.io/qt-5/qwidget.html#activateWindow?